selfdrive/car/tests/test_models.py

```diff
@@ -201,6 +201,42 @@ def test_panda_safety_rx_valid(self):
           self.assertTrue(self.safety.addr_checks_valid())
     self.assertFalse(len(failed_addrs), f"panda safety RX check failed: {failed_addrs}")
 
+  def test_panda_safety_tx_cases(self, data=None):
+    """Asserts we can tx common messages"""
+    if self.CP.notCar:
+      self.skipTest("Skipping test for notCar")
```

Comment:
why are not cars skipped?

Reply:
openpilot and safety both assume that `controls_allowed` will always effectively be true. if it's false, it blocks all messages, even zero torque ones, so we need to fix that first before enabling